crates/ui/src/slider.rs

```diff
@@ -123,6 +123,11 @@ impl SliderValue {
     }
 }
 
+pub enum SliderScale {
+    Linear,
+    Logarithmic,
+}
```

Comment:
The was default value is `Logarithmic` right?

Here need impl `Default`, and `SliderState` new should be `SliderScale::default()`.

Reply:
The previous behaviour was `Linear`, so I think that should be the default value

Reply:
Added the default and various other derives to SliderScale